fineract-provider/src/main/java/org/apache/fineract/portfolio/client/service/ClientReadPlatformServiceImpl.java

```diff
@@ -225,6 +228,7 @@ private String buildSqlStringFromClientCriteria(final SearchParameters searchPar
 
         String extraCriteria = "";
```

Comment:
This use case is legacy and used by some customers. So we can't get rid of this fix. 
So we have discussed internally before suggesting this fix.